examples/01_window/resources/config.ron

```diff
@@ -1,10 +1,11 @@
-# ---
-# display_config:
-dimensions: null
-fullscreen: false 
-max_dimensions: null
-min_dimensions: null
-multisampling: 1
-title: "Windowing example"
-visibility: true
-vsync: true
+
+DisplayConfig (
```

Comment:
The recommended RON formatting doesn't have any spaces between `DisplayConfig` and `(`. Also, I think it's clear this is the display config, so we could skip the type.